btree/btree.c

```diff
@@ -2403,15 +2408,19 @@ static void bnode_put(struct node_op *op, struct nd *node)
 		 */
 		ndlist_tlist_del(node);
 		ndlist_tlist_add(&btree_lru_nds, node);
+		lru_space_used += (m0_be_chunk_header_size() + node->n_size);
```

Comment:
Same comment as above.

Reply:
Root node will be put in the lru list only by destroy or close tick and mostly it's node descriptor will be removed from lru list and freed up immediately by node_put code(new code added for root node). So do we still need check for root node here?